src/controllers/v1/notificationsController.ts

```diff
@@ -167,20 +68,45 @@ export class NotificationsController {
     }
   }
 
+  @Post('/thirdParty/notificationsBulk')
+  @Security('basicAuth')
+  public async sendBulkNotification(
+    @Body()
+    body: sendBulkNotificationRequest,
+    @Inject()
+    params: {
+      microService: string;
+    },
+  ): Promise<SendNotificationResponse> {
+    const { microService } = params;
+    try {
+      validateWithJoiSchema(body, sendBulkNotificationValidator);
+      await Promise.all(
```

Comment:
What will happen if two events have the same trackId? We have made that field unique in the schema, so normally it must throw an error! but the problem is that some notifications will be created and some not!
@mohammadranjbarz Can you write a test to examine it? 
If you agree with to change it, the best way in my view is to put all tackIds in a `Set` and compare its size with the notifications size.

Reply:
Note, only duplicates in current notifications may cause a problem. If we have a trackId saved before this request, it will only gracefully ignored.

Reply:
Thanks @aminlatifi , your suggestion was good, I did it.